src/main/scala/org/constellation/gossip/state/GossipMessage.scala

```diff
@@ -25,6 +26,6 @@ case class GossipMessage[A](
 }
 
 object GossipMessage {
-  implicit def gossipMessageEncoder[A: Encoder]: Encoder[GossipMessage[A]] = deriveEncoder
-  implicit def gossipMessageDecoder[A: Decoder]: Decoder[GossipMessage[A]] = deriveDecoder
+  implicit def gossipMessageEncoder[F[_], A]: EntityEncoder[F, GossipMessage[A]] = BinaryCodec.encoder[F, GossipMessage[A]]
```

Comment:
Is it sent as a binary? I mean, does it work the same like we send snapshot info?

Reply:
It is sent as binary, but we don't need to convert objects to byte-array explicitly in the service layer, like it's done with Snapshot Infos. 

I tested it on a cluster and it does solve the signature validation issue, that I was talking about.